rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParserOnlyContribution.java

```diff
@@ -138,6 +138,11 @@ public InterruptibleFuture<ISet> lenses(ITree input) {
         return InterruptibleFuture.completedFuture(VF.bool(false));
     }
 
+    @Override
+    public CompletableFuture<IList> parseCodeActions(String commands) {
+        return CompletableFuture.completedFuture(null);
```

Comment:
don't return null, return `VF.list()`. In most places in `rascal-lsp` we try really hard to avoid `null`s.